packages/mobile/src/screens/favorites-screen/LibraryCategorySelectionMenu.tsx

```diff
@@ -103,22 +111,23 @@ export const LibraryCategorySelectionMenu = () => {
     <View style={styles.container}>
       <ScrollView
         horizontal
-        role='radiogroup'
         accessibilityRole='radiogroup'
         contentContainerStyle={styles.scrollContainer}
         alwaysBounceHorizontal={false}
       >
-        {categories.map((c) => (
-          <HarmonySelectablePill
-            key={c.value}
-            accessibilityRole='radio'
-            aria-checked={selectedCategory === c.value}
-            onPress={() => handleClick(c.value)}
-            role='radio'
-            label={c.label}
-            isSelected={selectedCategory === c.value}
-          />
-        ))}
+        {categories.map((category) => {
+          const { value, label } = category
+          return (
+            <SelectablePill
```

Comment:
nit: While I do generally prefer destructuring, it does feel a bit weird to me to do it inline in the render, ngl